classes/item.rb

```diff
@@ -12,6 +12,11 @@ def initialize(publish_date, archived)
     @genre = nil
```

Comment:
Hey @joskalenda, great job so far!. Kindly initialise @label to nil in the initialise block

Reply:
👍